course/urls.py

```diff
@@ -4,5 +4,7 @@
 
 urlpatterns = [
     path('catalog/', views.get_catalog, name='catalog'),
-    path('catalog/categories', views.getCourseCategories, name='catalog'),
+    path('catalog/categories', views.get_course_categories, name='categories'),
+    path('catalog/categories/<int:category_id>',
+         views.get_category_courses, name='category-courses')
```

Comment:
let's rename this method to something more clear, such as "get_courses_for_category"